### PR TITLE
Add quick links to job types in batch

### PR DESCRIPTION
**Pull request by cassimons**
Another QOL change. Adds quick links to filter jobs by status. Just easier than having to type in text box for common actions.

I have not actually tested this renders as expected.Is it easy to run up a local instance?

### CODE REVIEW

**Comment by lgruen** [comment]
Thanks! We can try this in a `dev deploy`, which unfortunately requires full Hail admin permissions.

@illusional, did you already commit your changes to get your recent `dev deploy` working? Then I could clone this branch and start a deploy myself.

File: batch/batch/front_end/templates/batch.html [hunk 1]
```diff
@@ -15,10 +15,11 @@ <h2>Properties</h2>
...
   <ul>
-    <li>Pending Jobs: {{ batch['n_jobs'] - batch['n_completed'] }}</li>
-    <li>Succeeded Jobs: {{ batch['n_succeeded'] }}</li>
-    <li>Failed Jobs: {{ batch['n_failed'] }}</li>
-    <li>Cancelled Jobs: {{ batch['n_cancelled'] }}</li>
+    <li><a href="{{ base_path }}/batches/{{ batch['id'] }}?q=pending">Pending</a>
+      + <a href="{{ base_path }}/batches/{{ batch['id'] }}?q=running">Running</a> Jobs: {{ batch['n_jobs'] - batch['n_completed'] }}</li>
+    <li><a href="{{ base_path }}/batches/{{ batch['id'] }}?q=success">Succeeded</a>  Jobs: {{ batch['n_succeeded'] }}</li>
+    <li><a href="{{ base_path }}/batches/{{ batch['id'] }}?q=failed">Failed</a> Jobs: {{ batch['n_failed'] }}</li>
+    <li><a href="{{ base_path }}/batches/{{ batch['id'] }}?q=cancelled">Cancelled</a>  Jobs: {{ batch['n_cancelled'] }}</li>
```
Comment (lgruen) [inline batch/batch/front_end/templates/batch.html:22]:
```suggestion
    <li><a href="{{ base_path }}/batches/{{ batch['id'] }}?q=success">Succeeded</a> Jobs: {{ batch['n_succeeded'] }}</li>
    <li><a href="{{ base_path }}/batches/{{ batch['id'] }}?q=failed">Failed</a> Jobs: {{ batch['n_failed'] }}</li>
    <li><a href="{{ base_path }}/batches/{{ batch['id'] }}?q=cancelled">Cancelled</a> Jobs: {{ batch['n_cancelled'] }}</li>
```

**Review by lgruen — APPROVED** [review]
Looks good to me and the links are working fine:
CleanShot 2023-03-02 at 23 12 16@2x

**Comment by cassimons** [comment]
Thanks Leo, that looks good. Am I safe to merge this one?

**Pull request merged**